packs/spells/cascade-countermeasure.json

```
@@ -11,7 +11,7 @@
         "damage": {},
         "defense": null,
         "description": {
-            "value": "<p>You quickly adjust your @UUID[Compendium.pf2e.actionspf2e.Item.Arcane Cascade] to offer magical protection. You gain resistance 5 against damage from spells. As normal, using Arcane Cascade again means you've ended the stance, and the spell ends.</p>\n<hr />\n<p><strong>Heightened (+3)</strong> The resistance increases by 5.</p>\n<p data-visibility=\"owner\"><em>Note: Conflux spells refresh Spellstrike.</em></p>"
+            "value": "<p>You quickly adjust your @UUID[Compendium.pf2e.actionspf2e.Item.Arcane Cascade] to offer magical protection. You gain resistance 5 against damage from spells. As normal, using Arcane Cascade again means you've ended the stance, and the spell ends.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Cascade Countermeasure]</p>\n<hr />\n<p><strong>Heightened (+3)</strong> The resistance increases by 5.</p>\n<p data-visibility=\"owner\"><em>Note: Conflux spells refresh Spellstrike.</em></p>"
```

```suggestion
            "value": "<p>You quickly adjust your @UUID[Compendium.pf2e.actionspf2e.Item.Arcane Cascade] to offer magical protection. You gain resistance 5 against damage from spells. As normal, using Arcane Cascade again means you've ended the stance, and the spell ends.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Cascade Countermeasure]</p>\n<hr />\n<p><strong>Heightened (+3)</strong> The resistance increases by 5.</p>\n<em>Note: Conflux spells refresh Spellstrike.</em>"
```
We may consider adding some item alterations to do those conflux spell notes